liana/src/config.rs

```diff
@@ -322,7 +312,6 @@ mod tests {
         {
             let toml_str = r#"
             data_dir = '/home/wizardsardine/custom/folder/'
-            daemon = false
```

Comment:
I think you could now remove the `#[cfg(unix)]` attribute above and elsewhere in this test.